typings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "dependencies": {
+    "chai": "registry:npm/chai#3.5.0+20160415060238",
+    "immutable": "registry:npm/immutable#3.7.6+20160411060006"
+  },
+  "globalDevDependencies": {
+    "mocha": "registry:env/mocha#2.2.5+20160321223601"
+  }
+}
```

Comment:
This is a bit cryptic to me, what does that mean when I bump dependency versions?
Also, there seems to be a discrepancy between versions here and in `package.json`.

In general, anything in particular I need to know when I add features to the API, publish a version, bump a dep, ...?